packages/ui/src/domainModules/locality/components/LocalityManager/item/EditForm.js

```diff
@@ -40,6 +42,7 @@ export class Edit extends PureComponent {
           formSectionNavigationSubHeader={capitalizeFirstLetter(
             initialValues.group
           )}
+          formValueSelector={this.formValueSelector}
```

Comment:
`this.formValueSelector` doesn't exist in this file. do you need it?

Reply:
It needed in recordHistoryEvents